demos/pw-react-app/yarn.lock

```diff
@@ -1068,9 +1068,8 @@
   "resolved" "https://registry.npmjs.org/@nodelib/fs.stat/-/fs.stat-1.1.3.tgz"
   "version" "1.1.3"
 
-"@playwire/pw-react-component@^1.0.7":
-  "integrity" "sha512-lv8ftztSjlLt7lAEF4/z9kMoT2oCISLQxsgBFeTEmg4xGyaH2PvgryOyRZLoKhJtcZai/3jts7IaipPOACMgbA=="
-  "resolved" "https://registry.npmjs.org/@playwire/pw-react-component/-/pw-react-component-1.0.7.tgz"
+"@playwire/pw-react-component@file:../..":
```

Comment:
Is it ok for our case - having an URL to local file instead of repository URL?

Reply:
Correct, that is only for testing during code review, I will update it when publishing it to npm registry, @vitkrv @tcreamer1